test/test_files/tinysnb/algorithm/basic.test

```diff
@@ -3,7 +3,17 @@
 --
 
 -CASE BasicAlgorithm
--STATEMENT CALL variable_length_path(graph("person", "knows"), 1, 1) RETURN *;
+
+-STATEMENT PROJECT GRAPH PK (person (age), knows) CALL variable_length_path(PK, 1, 1) RETURN *;
```

Comment:
Should we change the inner ( ) where we specify the projected properties with { }? It would be better to use different delimiters and may make it closer to the { } we use in node and rel patterns in MATCH clauses, e.g., (a:Person { age: 25}). We also use { } to project intermediate properties in var-len rel patterns: e.g., `(a:User)-[e:Follows*1..2 (r, n | WHERE r.since > 2020 | {r.since}, {n.name})]`.